api/src/queries/dwc/dwc-queries.ts

```diff
@@ -0,0 +1,629 @@
+import { SQL, SQLStatement } from 'sql-template-strings';
+import { getLogger } from '../../utils/logger';
+
+const defaultLog = getLogger('queries/dwc/dwc-queries');
+
+/**
+ * SQL query to get submission occurrence record given package ID for a particular survey.
+ *
+ * @param {number} dataPackageId
+ * @returns {SQLStatement} sql query object
+ */
+export const getSurveyOccurrenceSubmissionSQL = (dataPackageId: number): SQLStatement | null => {
```

Comment:
Small nit: this function (and most of them I believe) dont ever return `null`.  So the return type could be updated to just say `SQLStatement`

```suggestion
export const getSurveyOccurrenceSubmissionSQL = (dataPackageId: number): SQLStatement => {
```

Reply:
adjusted